ftw/keywordwidget/profiles/default/metadata.xml

```diff
@@ -1,4 +1,3 @@
 <?xml version="1.0"?>
 <metadata>
-    <version>1000</version>
```

Comment:
Oh ok, I'll remove the registration of the upgrade step directory for the default profile in this case.